homeassistant/components/adax/config_flow.py

```diff
@@ -0,0 +1,81 @@
+"""Config flow for Adax integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+import adax
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+
+from .const import ACCOUNT_ID, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {vol.Required(ACCOUNT_ID): int, vol.Required(CONF_PASSWORD): str}
+)
+
+
+async def validate_input(hass: HomeAssistant, data: dict[str, Any]) -> None:
+    """Validate the user input allows us to connect."""
+    account_id = data[ACCOUNT_ID]
+    password = data[CONF_PASSWORD].replace(" ", "")
+
+    for entry in hass.config_entries.async_entries(DOMAIN):
+        if entry.data[ACCOUNT_ID] == account_id:
+            raise AlreadyConfigured
```

Comment:
```suggestion
    self._async_abort_entries_match({ACCOUNT_ID: account_id})
```

This can probably be moved directly into the user step method, above the try block. That way you don't need (and can remove) the AlreadyConfigured class as well.